examples/native/native_test.c

```diff
@@ -323,15 +326,27 @@ int TPM2_Native_TestArgs(void* userCtx, int argc, char *argv[])
 
 
     /* Random */
-    XMEMSET(&cmdIn.getRand, 0, sizeof(cmdIn.getRand));
-    cmdIn.getRand.bytesRequested = MAX_RNG_REQ_SIZE;
-    rc = TPM2_GetRandom(&cmdIn.getRand, &cmdOut.getRand);
+#if defined(WOLFTPM_ST33) || defined(WOLFTPM_AUTODETECT)
```

Comment:
It appears not all ST supports this. For that case if we see `TPM_RC_COMMAND_CODE` I think it should fall back to `TPM2_GetRandom` so this test can still pass on that hardware.